app/models/user_account.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class UserAccount < ApplicationRecord
+  has_many :form_submissions, dependent: :nullify
```

Comment:
I think we'll still want the historical record of form submissions for any given user, if they had an account in the past. Just for context, the use case that I am personally interested in this `form_submissions` model for has to do with resubmitting failed form submissions. And since we have some unauthenticated user form submissions, I'd rather not needlessly lose form submissions from the past just because an account was deleted (in case I need to trigger a resubmit).